recipes/aws-c-event-stream/all/conanfile.py

```diff
@@ -74,6 +74,8 @@ def package(self):
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "aws-c-event-stream")
         self.cpp_info.set_property("cmake_target_name", "AWS")
+        self.cpp_info.names["cmake_find_package"] = "AWS"
```

Comment:
it seems to me that there should be more attribute restored ? https://github.com/conan-io/conan-center-index/commit/fafc706b11a66f0af96da1a01e0f4d60a3418451#diff-b2bdc2286731f0ad8f0e17b33a876a8cf27bce1cbd8e7fbb6b32c18cdad4fc49L75